cmd/showbudget/show_budget_test.go

```diff
@@ -17,12 +18,14 @@ import (
 var _ = gc.Suite(&showBudgetSuite{})
 
 type showBudgetSuite struct {
+	coretesting.FakeJujuXDGDataHomeSuite
 	testing.CleanupSuite
```

Comment:
Because you're embedding two suites, you'll need to make (SetUp|TearDown)(Test|Suite) call both suites, otherwise they'll be ignored.